Eastern-Research-Group/mywaterway
# Feature/158 switch to ordspub

## Related Issues:
* https://jira.epa.gov/browse/HMW-158

## Main Changes:
* Switched from ofmpub to ordspub. I also added this URL to the services.json file.
* Synced up the services.json and services-attains.json files.
* Updated some comments to reference the correct services. Looked like ofmpub was copied/pasted a bunch of times. 
* Updated the Google Analytics mapping since ofmpub was incorrectly tied to other services. Likely a copy/paste issue.

## Steps To Test:
1. Navigate to http://localhost:3000/community/dc/drinking-water
2. Click on the "EPA's Safe Drinking Water Information System (SDWIS) Federal Reporting Services" link
3. Verify it opens the same page as in production
4. Expand an item in the list
5. Click on the "More details" link
6. Verify it opens the same page as in production
7. Click on the "Who withdraws water for drinking here?"
8. Repeat steps 4 - 6
9. Navigate to http://localhost:3000/state/AL/water-quality-overview
10. Click on the "Drinking Water" tab
11. Click on the "View detailed drinking water data for Alabama" link
12. Verify it opens the same page as in production

## TODO:
